pandas/core/sorting.py

```diff
@@ -385,7 +385,7 @@ def ensure_key_mapped(values, key: Optional[Callable], levels=None):
     from pandas.core.indexes.api import Index
 
     if not key:
-        return values.copy()
+        return values
```

Comment:
This change doesn't seem related. Is it coming from manually fixing a commit?

Reply:
yes, sorry for that